modules/python/package/cv2/__init__.py

```diff
@@ -179,3 +203,13 @@ def load_first_config(fnames, required=True):
 
 
 bootstrap()
+
+# hotfix for pylint issue (https://github.com/opencv/opencv-python/issues/570)
+from . import getStructuringElement, MORPH_ELLIPSE
+
+# extra imports for a proper autocomplete work in IDE
+from . import _registerMatType
+from . import gapi
+from . import mat_wrapper
+from . import misc
+from . import utils
```

Comment:
Okay. I would only suggest to keep it here, which fixes pylint issue (it even happens if to build opencv from the source) and allows to use modules (e.g. `gapi`, `misc`) using autocomplete in IDE.

Reply:
Unfortunately `pylint` can't normally work with modern `cv2` without manual own "brain" (like numpy) or plugin module (like django).
It doesn't see anything except `import_module("cv2.cv2")` line in `opencv_python` package.